### ISSUE TITLE
Typing text in action sequence

### ISSUE THREAD

**Issue by XelorR** [k0kubun/xremap#323]
Hello,

is it possible to type text in sequence of key actions?

Autohotkey 2.0 working example:
`!,::Send "{Ctrl Down}{t}{Ctrl Up}chrome://settings{Enter}"`

It would be nice to make something like this possible:
` Alt-comma: [Control-t, "chrome://settings", Enter]`

**Comment by mritunjaymusale** [comment]
+1 this feature request

**Comment by k0kubun** [comment]
Interesting idea. Since `a` and `"a"` are indistinguishable in YAML, I would make it something like `Alt-comma: [Control-t, {text: chrome://settings}, Enter]`. Would any of you be interested in working on a pull request for it?

**Comment by XelorR** [comment]
> Interesting idea. Since `a` and `"a"` are indistinguishable in YAML, I would make it something like `Alt-comma: [Control-t, {text: chrome://settings}, Enter]`. Would any of you be interested in working on a pull request for it?

Agree, something like `{text: chrome://settings}` is nice and clearly understandable in config file.

I'd like to help somehow but unfortunately I am not familiar with Rust and not sure where to start.
